lib/Settings/Settings.h

```diff
@@ -59,7 +59,7 @@ enum SettingType {
 class Settings {
  public:
   Settings()
-      : deviceName(String(("rfESP_")) + String(ESP.getChipId(), HEX)),
+      : deviceName("mqtt-433-gateway"),
```

Comment:
For compatibility with previous versions, the default device name should be `rf434`.